packages/styles/scss/components/card/_card.scss

```
@@ -166,6 +166,17 @@
     }
   }
 
+  :host(#{$c4d-prefix}-link-list-item-card),
+  :host(#{$c4d-prefix}-link-list-item-card-cta) {
```

```suggestion
  :host(#{$c4d-prefix}-link-list-item-card) {
```
Since `card-cta` has been changed